main.go

```diff
@@ -48,6 +48,8 @@ func main() {
 	getRename := getFlags.String("r", "", "The -r flag instructs to get existing binary and rename it with given name."+
 		" Allowed characters [A-z0-9._-]. If -r is used and no package/binary is specified or non existing binary name is used, bingo"+
 		" will return error. Cannot be used with -n.")
+	getDownloadMode := getFlags.String("m", "", "The -m flag instructs the go build command which go module download mode to use."+
```

Comment:
Let's remove this flag and do `mod` directly. This is what we aimed for designing Bingo (: 

For vendor mod in order to allow https://github.com/bwplotka/bingo/issues/41, if any we would need to create separate vendor dirs or prepare some cache dirs, so we will need something more custom anyway (: